Source/WebCore/testing/Internals.cpp

```diff
@@ -1425,7 +1425,7 @@ ExceptionOr<String> Internals::shadowRootType(const Node& root) const
 
 const AtomString& Internals::userAgentPartId(Element& element)
```

Comment:
Can we rename this method and its setter for consistency to pseudo() and setPseudo()?

Reply:
It's just a mass rename that needs to be done in LayoutTests/, shouldn't be too hard.